integration/spark/src/main/scala/org/apache/carbondata/indexserver/IndexJobs.scala

```diff
@@ -89,7 +89,7 @@ class DistributedIndexJob extends AbstractIndexJob {
         }
         client.getSplits(indexFormat)
           .getExtendedBlocklets(indexFormat.getCarbonTable.getTablePath, indexFormat
-            .getQueryId, indexFormat.isCountStarJob)
+            .getQueryId, indexFormat.isCountStarJob, null)
```

Comment:
Do not change this, but add a new `getExtendedBlocklets` function with one more param

Reply:
refactored